docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -469,9 +469,56 @@ Requested test runs:
 Subnational Approach
 ~~~~~~~~~~~~~~~~~~~~
 
+In order to include SQ-LNS targeting by location, we are switching to 
+use a subnational approach for most data in Wave 3. However, rather 
+than model all subnational locations separately, simulants 
+will just be assigned to a subnational location within their primary 
+location, and have input data pulled for the subnational location instead. 
+Unless otherwise specified in the model request table below, the data outputs 
+do not need to stratified by subnational location.
+
+
+Initializing Simulant's Locations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Simulants will be obtained from the pregnancy sim, the same as in prior 
+waves. These simulants will already have a country location. The pregnancy 
+simulation is only run at the national level.
+
+When these simulants are loaded into the child simulation, they will be 
+assigned a subnational location within their country. Here is the data 
+for the `percent of simulants assigned to each subnational location by sex <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/ff08145109e1434669f08afe702ffc5e3d45a6c2/data_prep/sqlns_subnational/subnational_percents.csv>`_. Note that all 3 countries are included 
+in this csv file. 
+
+Since LBWSG is done nationally, there is a risk in assigning locations 
+at birth that they will be incorrect by 6 months of age. Subnational 
+locations with higher LBWSG should have more deaths, which will not be captured here. 
+However, after reviewing the changes in population distribution between 
+birth and 6 months in GBD data, we found this impact to be very minimal, and 
+therefore believe this is a reasonable limitation. Additionally, 
+overall mortality will still be at a subnational level, further 
+mitigating these effects. 
+
+Data Inputs: 
+^^^^^^^^^^^^
+
+Once a simulant is assigned to a subnational location, most GBD data used 
+will be subnational specific data. LBWSG, which comes as an output from the 
+pregnancy sim, will be national. Similarly, the PAFs for LBWSG will also 
+be national. All other GBD data will be at the subnational level.
+
+Artifacts will be made for all subnational 
+geographies. We will also regenerate data for all custom made datasets, 
+such as wasting transitions, PAFs, and CGF correlation, at the subnational 
+level. 
+
+SAM and MAM treatmet coverage and efficacy data will continue to be national only. Also, 
```

Comment:
Let's brainstorm strategies to go beyond this assumption.  I think it could be a valuable addition to our model, and the data for Ethiopia might be included in the stuff that Latera has been calling our attention to (which I have now formally requested).

Reply:
We are also using a summary national-level value for coverage from the Isanaka paper that we are currently using as our data source. we won't have full geographic coverage, but we might be able to glean something from that data source. Overall though, I expect modeling this at the subnational level (except maybe for Ethiopia?) will be difficult due to data availability.